clang/tools/amdgpu-arch/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@
 
 set(LLVM_LINK_COMPONENTS Support)
 
-add_clang_tool(amdgpu-arch AMDGPUArch.cpp AMDGPUArchByHSA.cpp AMDGPUArchByHIP.cpp)
+add_clang_tool(amdgpu-arch DISABLE_CLANG_LINK_DYLIB
```

Comment:
Why is this not allowed? Can we add a note about this?

Reply:
There using target_link_libraries instead of clang_target_link_libraries so there not linked to clang-cpp so we have to disable import macros.